core/src/allocation_optim.jl

```diff
@@ -6,45 +6,12 @@ function add_objective_term!(
     ex::Union{JuMP.QuadExpr, JuMP.AffExpr},
     F_variable::JuMP.VariableRef,
```

Comment:
Also here there are obsolete inputs. However, here you don't have to take that much care because this function falls under a planned refactor: https://github.com/Deltares/Ribasim/issues/1196.

Reply:
I have removed these unused inputs